mention default transforms (#139)
OpenIndiana/oi-docs: docs/dev/userland.md

to generate a manifest from the list of installed files.

Copy the file `manifests/sample-manifest.p5m` to `$(COMPONENT_NAME).p5m` and edit it:

* Add your name as a contributor
* Remove unused entries from the manifest:
    * directories: `:%g/^dir/d` (Vim)
    * static libs: `:%g/.a$/d` (Vim)
    * libtool files: `:%g/.la$/d` (Vim)
    * Python *.pyc: `:%g/.pyc$/d` (Vim)

For some components, specific rules need to be applied: they can be implemented with *transforms*.

> A default set of transforms that are applied to each package during the publishing process can be found in the `transforms` directory at the root of oi-userland.  The default transforms in that directory can be used as examples if you need to add custom rules for your component.

Reviewer: 1) This is not entirely correct. Transforms are applied based on Makefile rules. Look at make-rules/ips.mk. Some transforms can be applied based on different conditions, for example, if auto-generated manifest is used.
2) Each sentence should be on separate line.

Author: Thanks @pyhalov .   I misunderstood Michal's response to the question I asked on oi-dev about transforms.    I'll update the changes in my patch to be more precise and push new changes.

to generate a manifest from the list of installed files.

Copy the file `manifests/sample-manifest.p5m` to `$(COMPONENT_NAME).p5m` and edit it:

* Add your name as a contributor
* Remove unused entries from the manifest:
    * directories: `:%g/^dir/d` (Vim)
    * static libs: `:%g/.a$/d` (Vim)
    * libtool files: `:%g/.la$/d` (Vim)
    * Python *.pyc: `:%g/.pyc$/d` (Vim)

For some components, specific rules need to be applied: they can be implemented with *transforms*.

> Some transforms, located in the directory `transforms` at the root of oi-userland, are applied automatically by various Makefile targets defined in `make-rules/ips.mk`.

Reviewer 2: ```suggestion
Transforms, located in the directory `transforms` and in various include files in the `make-rules` directory at the root of oi-userland, are applied automatically by various Makefile targets.
```

Reviewer: Transforms are not located in "various include files in the `make-rules` directory", needs better wording.
